website/docs/cloud-docs/api-docs/stacks.mdx

```diff
@@ -944,3 +952,476 @@ This endpoint redirects to the download URL for a stack configuration source bun
 | [302][] | HTTP Redirect             | Successfully redirect client to temporary source bundle download URL  |
 | [404][] | [JSON API error object][] | Stack configuration not found, or user unauthorized to perform action |
 
+## List stack deployments
+
+This endpoint lists all the deployments under the specified stack.
+
+`GET /stacks/:stack_id/stack-deployments`
+
+| Parameter   | Description                                  |
+|-------------|----------------------------------------------|
+| `:stack_id` | The id of the stack to list deployments for. |
+
+### Query Parameters
+
+This endpoint supports pagination [with standard URL query parameters](/terraform/cloud-docs/api-docs#query-parameters); remember to percent-encode `[` as `%5B` and `]` as `%5D` if your tooling doesn't automatically encode URLs.
+
+| Parameter      | Description                                                                 |
+|----------------|-----------------------------------------------------------------------------|
+| `page[number]` | **Optional.** If omitted, the endpoint will return the first page.          |
+| `page[size]`   | **Optional.** If omitted, the endpoint will return 20 policy sets per page. |
+
+### Sample Request
+
+```shell
+$ curl\
+  --header "Authorization: Bearer $TOKEN" \
+  --header "Content-Type: application/vnd.api+json" \
+  https://app.terraform.io/api/v2/stacks/st-LSqektntTNJrAhCt/stack-deployments
+```
+
+### Sample Response
+
+/* TODO: For reviewer - check if stack-deployment-runs should belong in the links as you need the stack deployment group id to fetch it */
```

Comment:
Can we leave this todo comment up? I think we're not going to render the runs here but I'm sure that's up in the air to some degree so we can double check when the apis are more finalized.